translations/dde-store_zh_CN.ts

```diff
@@ -0,0 +1,435 @@
+        <source>Notifications</source>
+        <translation>通知</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="89"/>
+        <source>Show non-app updates as</source>
+        <translation>没有软件更新时显示</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="90"/>
+        <source>Max apps per list</source>
+        <translation>列表最大软件数</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="91"/>
+        <source>Minimize to tray on exit</source>
+        <translation>退出时最小化到托盘</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="92"/>
+        <source>Check for updates</source>
+        <translation>检查更新频率</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="93"/>
+        <source>App installed</source>
+        <translation>安装完软件</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="94"/>
+        <source>App uninstalled</source>
+        <translation>卸载完软件</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="95"/>
+        <source>Available updates</source>
+        <translation>更新就绪</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="96"/>
+        <source>Finished updates</source>
+        <translation>更新完成</translation>
+    </message>
+</context>
+<context>
+    <name>SnapPlugin</name>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="112"/>
+        <source>Installed &quot;%1&quot;</source>
+        <translation>已安装 %1</translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="150"/>
+        <source>Uninstalled &quot;%1&quot;</source>
+        <translation>已卸载 %1</translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="218"/>
+        <source>Classic mode required</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="218"/>
+        <source>This snap requires confinement to be disabled via classic mode</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="220"/>
+        <source>Cancel</source>
+        <translation>取消</translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="221"/>
+        <source>Continue</source>
+        <translation>继续</translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="227"/>
+        <source>Snap Error</source>
+        <translation type="unfinished">Snap 出错</translation>
```

Comment:
The `type="unfinished"` needs to be removed from finished translations for those strings to show up

Reply:
[`type="unfinished"` is not deleted because I am not sure what the original text means.](https://translate.google.cn/?sl=auto&tl=en&text=%E6%B2%A1%E6%9C%89%E5%88%A0%E9%99%A4%20type%3D%22unfinished%22%20%E6%98%AF%E5%9B%A0%E4%B8%BA%E6%88%91%E8%BF%98%E4%B8%8D%E7%A1%AE%E5%AE%9A%E5%8E%9F%E6%96%87%E7%9A%84%E6%84%8F%E6%80%9D%E3%80%82&op=translate)